.semaphore/semaphore.yml

```diff
@@ -53,6 +53,7 @@ blocks:
     task:
       prologue:
         commands:
+          - sem-version ruby 2.5.7
```

Comment:
Semaphore uses `.ruby-version` file by default - https://docs.semaphoreci.com/article/73-ruby#selecting-a-ruby-version-on-linux. You do not need to select version here.